safe-global/safe-wallet-web
# fix: Use correct surplus symbol for buy orders

## What it solves

Resolves [Notion issue](https://www.notion.so/safe-global/Issue-Check-surplus-calculation-31c4ea517ab74a2a8c8f1555a6447fb1), [Notion issue](https://www.notion.so/safe-global/Issue-cancelled-state-looks-different-from-the-design-0289beee434b4fb18b29c9432846ebdc)

## How this PR fixes it

- Changes the cancelled icon and color according to the Design
- Uses the correct token symbol for the surplus
- Removes a bunch of logic that wasn't used from `SwapOrderConfirmationView`

## How to test it

1. Create and execute a buy order
2. Observe the same surplus symbol is displayed as in the cow explorer

## Screenshots

## Checklist
